BlockParty.py

```diff
@@ -202,7 +202,8 @@ def init_game(self):
             self.hscore = self.read_highscore()
         self.score = 0
         self.new_figure()
-        self.set_level(5)
+        if not hasattr(self, 'level') or self.level is None:
+            self.set_level(0)
```

Comment:
There's no need for the if statement here because this is the first place the level is set when the game starts.

Reply:
No reason for additional check though.